recipes/kraken-all/meta.yaml

```diff
@@ -0,0 +1,13 @@
+package:
+  name: kraken-all
+  version: "0.10.5beta"
+requirements:
+  build:
+    - kraken ==0.10.5beta
```

Comment:
I don't quite get this. Here, we have a strict build dependency, but in the run requirements, we depend on any kraken version?